apps/showcase/src/assets/trainings/sdk/steps/model-extension/instructions.html

```diff
@@ -0,0 +1,56 @@
+<p>
+  There are certain cases in which you want to be able to extend your SDK models, and therefore ensure that revivers are generated.
+</p>
+<p>
+  Let's continue with the use case of the previous exercise. In order to keep track of the user's current booking, it would be useful to generate an ID.
+  To do this, we can create a new model which extends the previously generated <code>Flight</code> type.
+</p>
+<p>
+  Before beginning the exercise, we need to make sure that the existing API was generated with model extension in the previous steps.
+  To verify this, check out the configuration used to generate the API (either the command line or the file <code>openapitools.json</code>).
+  Ensure that the global property <code>allowModelExtension</code> has been set to <b>true</b>. This will guarantee that the revivers of the
+  base models are generated, which is essential for the following exercise.
+</p>
+
```

Comment:
It is a bit weird - we say that they should run commands as it is essential for the following exercise but we do it for them.
Maybe, we should be clearer:

You will be working with an sdk which has been generated with the .... option etc.

This way we make it clearer that they will need to do it if they want a local run or use it in their project

Reply:
updated